packages/react/src/components/header/Header.stories.tsx

```diff
@@ -64,7 +64,7 @@ export const WithFullFeatures = (args) => (
           </Link>
         </Header.NavigationLanguageSelector>
 
-        <Header.ActionBarItem fullWidth label="Haku" icon={IconSearch} id="action-bar-search">
+        <Header.ActionBarItem fullWidth label="Haku" icon={<IconSearch />} id="action-bar-search">
```

Comment:
The icon has to be passed this way? How did it behave previously? I'm just curious.

Reply:
It didn't work. In other stories icons are used like this, but Header stories had just icon names...And I haven't seen those icons in Header maybe never. ...but I just checked how other components use those and used same pattern